indexing.go

```diff
@@ -123,7 +129,7 @@ func BulkIndex(docs []string, options Options) error {
 		// use it in the header.
 		if options.IDField != "" {
 			var docmap map[string]interface{}
-			dec := json.NewDecoder(strings.NewReader(doc))
+			dec := jsoniter.NewDecoder(strings.NewReader(doc))
```

Comment:
I understand and agree, I had some issues with jsoniter too, but I filed an issue that has fixed over nigth.
Any way I'll revert to native json